float64[] x_filt
float64[] u
float64[] u_lin
float64[] y
float64[] y_lin
